bin/parse_podfile_lock

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+
+require 'json'
+require 'yaml'
+
+dependency_metadata_regex = /(?<pod>.*?)\s\((=|(~>))?\s?(?<version>(\d*\.?)+)\)/
+
+podfile_lock = nil
+begin
+  podfile_lock = YAML.safe_load(File.read(ARGV[0]))
+rescue SyntaxError, Psych::SyntaxError
+  warn 'Unable to parse Podfile.lock file'
+  exit(1)
+end
+
+if podfile_lock['PODS'].nil? || podfile_lock['DEPENDENCIES'].nil?
+  warn '"PODS" or "DEPENDENCIES" key not found in Podfile.lock!'
+  exit(1)
+end
+
+dependencies = []
+# Parse individual dependency entry
```

Comment:
My understanding is that the bodies of the both loops are same. Can we write it in the following way? Correct me if I am wrong.

```
(podfile_lock['PODS'] + podfile_lock['DEPENDENCIES']).each do |dependency_entry|
  dependency_properties = dependency_metadata_regex.match(dependency_entry)
  dependency_hash = {}
  dependency_hash[:pod] = dependency_properties[:pod]
  dependency_hash[:version] = dependency_properties[:version]
  dependencies.append(dependency_hash)
end
```